DiffPlex/DiffBuilder/SideBySideDiffBuilder.cs

```diff
@@ -104,6 +104,59 @@ public static SideBySideDiffModel Diff(IDiffer differ, string oldText, string ne
             return model;
         }
 
+        public static SideBySideDiffModel Diff(
+            IDiffer differ, 
+            string oldText, string newText,
+            List<IChunker> detailsPack,
```

Comment:
I like the idea to give an array, but detailsPack name is too general. maybe called it chunkers or something

Reply:
Oka then Chunkers it is.